static/js/src/latest-news.js

```diff
@@ -43,7 +43,8 @@
       article._embedded &&
       article._embedded["wp:featuredmedia"] &&
       article._embedded["wp:featuredmedia"][0] &&
-      article._embedded["wp:featuredmedia"][0].width
+      article._embedded["wp:featuredmedia"][0].media_details &&
+      article._embedded["wp:featuredmedia"][0].media_details.width
```

Comment:
This check was updated here, but below when defining the `articleImage` you're still referencing `      article._embedded["wp:featuredmedia"][0].width` and       article._embedded["wp:featuredmedia"][0].height` was this intentional?

Reply:
There should be a height if there's a width but you're right, we don't know for sure